### PR TITLE
chore: pare down lib-jessie to only what is useful

### PR DESCRIPTION
**Pull request by michaelfig**
Let's start minimally!

### CODE REVIEW

**Review by dckc — COMMENTED** [review]
LGTM, though I'd rather not speak for @erights .

This leaves the `harden` dependency implicit; that's on purpose, right? Do we want `ses` among its `peerDependencies`?

File: lib/index.js [hunk 1]
```diff
@@ -2,9 +2,4 @@
 //
```
Comment (erights) [inline lib/index.js:2]: How much of this comment is still appropriate?

**Review by erights — APPROVED** [review]
LGTM! Onward!

File: package.json [hunk 2]
```diff
@@ -19,9 +19,6 @@
     "url": "https://github.com/Agoric/Jessie/issues"
   },
   "homepage": "https://github.com/Agoric/Jessie#readme",
-  "dependencies": {
-    "@agoric/harden": "0.0.4"
```
Comment (dckc) [inline package.json:23]: `makers.js` still has `import '@agoric/harden'`; is that on purpose?

Comment (michaelfig) [reply package.json:23]: Accident.  I've removed it.

File: lib/index.js (hunk 1 above)
Comment (michaelfig) [reply lib/index.js]: Pruned.

**Pull request merged**